hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -447,11 +448,17 @@ public static DataStreamSink<Object> dummySink(DataStream<Object> dataStream) {
   }
 
   public static String opName(String operatorN, Configuration conf) {
-    return operatorN + ": " + conf.getString(FlinkOptions.TABLE_NAME);
+    return operatorN + ": " + getDbTableName(conf);
   }
 
   public static String opUID(String operatorN, Configuration conf) {
-    return "uid_" + operatorN + "_" + conf.getString(FlinkOptions.TABLE_NAME);
+    return "uid_" + operatorN + "_" + getDbTableName(conf);
+  }
+
+  public static String getDbTableName(Configuration conf) {
```

Comment:
DATABASE_NAME was introduced in https://github.com/apache/hudi/pull/7682, but it was not set up automically like TABLE_NAME, we should set up it correctly first.

Reply:
Do you mean to add a new parameter in `FlinkStreamerConfig`? If not, could you help pointing out where the code is, thanks~

Reply:
In `HoodieTableFactory` we set up the table name based on the current table identifier/path. We can do similiar set up for database name.

Reply:
`getDbTableName` -> `getTableIdentifier`